webui/react/src/components/LogViewer/LogViewerFilters.settings.ts

```diff
@@ -10,48 +12,53 @@ export interface Settings {
   searchText?: string;
 }
 
-const config: SettingsConfig = {
-  settings: [
-    {
-      key: 'allocationId',
-      type: {
-        baseType: BaseType.String,
-        isArray: true,
-      },
+const config: SettingsConfig<Settings> = {
+  applicableRoutespace: 'log-viewer-filters',
+  settings: {
+    agentId: {
+      defaultValue: undefined,
+      storageKey: 'agentId',
```

Comment:
if we are sure that storageKey and key are always the same, we can remove `storageKey` cuz we might do typo in the future and spend lots of time to find out...

Reply:
It would be a nice optimization if storageKey defaulted to the same value as the key but let's leave that for a future PR